molo/profiles/forms.py

```diff
@@ -146,7 +147,8 @@ class EditProfileForm(forms.ModelForm):
     )
     date_of_birth = forms.DateField(
         widget=SelectDateWidget(
-            years=list(reversed([y for y in range(1930, datetime.now().year)]))
+            years=list(reversed([y for y in range(1930,
+                                datetime.now().year + 1)]))
```

Comment:
As a general rule, it's not a great idea to use `datetime.now()` in anything that does anything with timezones. In this case it's not that big a deal because you're only interested in the year, but it's probably still better to use something like `django.utils.timezone.now()`.